app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/dagger/StatsModule.kt

```diff
@@ -0,0 +1,12 @@
+package eu.quiqua.gloomhaven.statstracker.features.stats.dagger
+
+import dagger.Module
+import dagger.Provides
+
+@Module
+class StatsModule() {
+
+    @Provides
+    fun getModel(): String = "hello world"
```

Comment:
in our team we call these methods `providesXyz` - doesn't matter much though 🤷‍♀️